src/AppInstallerCommonCore/Public/winget/ManifestValidation.h

```diff
@@ -252,6 +253,13 @@ namespace AppInstaller::Manifest
     };
 
     static const std::unordered_set<std::filesystem::path> s_AllowedPortableFiletypes = { L".exe" };
+    static const std::unordered_set<std::filesystem::path> s_AllowedFontFiletypes = {

```

Comment:
I probably didn't see this during the portable review, but it would be much better if we didn't use a static that required dynamic allocation/destruction.  I would think that a `std::array<std::wstring_view>` should be able to be read-only memory (after loader fixups).

Also, we really shouldn't be defining this in the header.

Reply:
Fixed by changing to constexpr std::array<std::wstring_view, n> and putting into the anonymous namespace in ManifestValidation.cpp, for both font and portable filetypes, for read only memory with no more allocation/deallocation at runtime.